packages/paste-website/src/pages/experiences/artificial-intelligence/index.mdx

```diff
@@ -0,0 +1,351 @@
+- [Microsoft HAX toolkit](https://www.microsoft.com/en-us/haxtoolkit/)
+- [Microsoft HAX playbook](https://microsoft.github.io/HAXPlaybook/?state=xxxxxxxx) for simulating error scenarios
+
+### Building trust when introducing your feature
+
+When using Twilio AI features, customers care most about **data privacy** and the **outcomes** of a feature compared to the “old way” of doing things. When introducing your feature:
+
+- Include information about how customer data is handled, including how it’s used for training models, information on vendors the data is shared with, and how customers can minimize giving [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii) when it’s not needed. Taking these steps helps customers understand whether Twilio features meet their own AI policies.
+- Give customers the ability to compare AI-generated outcomes to their current experience.
+
+Based on your use case, you might also include information about how the AI model works and when customers’ data quality can improve AI feature performance.
+
+Since AI can power a wide range of features, customers can be unsure where to start. Use [Twilio’s generative AI design framework cards](https://www.figma.com/community/file/1392507403743352508/generative-ai-design-framework) and the [Shape of AI wayfinders](https://www.shapeof.ai/pattern-types/wayfinders) to prompt customers to the feature and describe it in a simple way, so that they can evaluate AI outcomes faster. When introducing the feature, let customers know:
+- How they can give feedback
+- How they can ask questions
+- What they should consider as they use the feature so they can ask the _right questions_, especially about their data and generated outcomes.
+
+### Naming your feature
+
+While AI features are still in rapid development, work with your Product Marketing team on a name. As a default, start with the name “[Product or Feature] assistant”.
+
+### Designing interactions and layouts
+
+Just because AI features are new, doesn’t mean we need to invent entirely new visual and interaction patterns for them. Start by asking yourself first: **“How would I design this feature if it did the same thing but didn’t use AI?”** Remember that customers have specific goals and tasks to achieve, and the task is rarely just “To use AI”. Avoid being too heavy-handed with communicating how a feature has been built or changed with AI since introducing new interaction patterns and unnecessary messaging can distract from the customer’s goals.
+
+<Blockquote>
+With AI-driven products, there can be a temptation to communicate the “newness” or “magic” of the system’s predictions through its UI metaphors.
+
+However, unfamiliar UI touchpoints can make it harder for users to learn to use your system, potentially leading to degraded understanding of, or trust in, your product…Instead, anchor new users with familiar UI patterns and features.
```

Comment:
issue: idk why these show up as different text sizes, but they should use the larger text size. You can combine it into one single paragraph if needed.